Add sample alertmanager configs and move cloudwatch exporter secrets

aditya-konarde opened the pull request:
Add an expected config+secret to get alertmanager running
Move cloudwatch exporter secret to secret.sample, ignore actual secret.

Commit:
Add sample alertmanager configs and move cloudwatch exporter secrets

Add an expected config+secret to get alertmanager running
Move cloudwatch exporter secret to secret.sample, ignore actual secret.

aditya-konarde commented:
@jfchevrette what do you think about this approach regarding the alertmanager config? 

The alertmanager secret is created from the 'alertmanager.config' file. So we can include either, or both here.

jfchevrette commented:
@aditya-konarde I think this is a good idea! I'd add a helper script somewhere in the repo that regenerates the secret from the config file content.  Maybe also add a comment to the top of the secret saying this is generated and that it should not be modified manually?

Commit:
Add comment about alertmanager secret

aditya-konarde commented:
@jfchevrette that's a good idea. For now I've added a comment at the top of the secret saying that its generated from the configfile. We'll need to add a utility script across multiple CR's, so I've added an issue to track that. 

Merging this PR to keep the repo in sync, and to get us going with the prometheus deployment
